File: problem-5/main.go
package main
import "fmt"

func main(){
	s := 1
	for {
		if isEvenlyDivisible(s){
			break
		}
		s++
	}
	fmt.Println(s)
}
func isEvenlyDivisible(n int) bool {
	for i := 1; i < 21; i++ {
		if n % i != 0 {
			return false
		}
	}
	return true
}

File: problem-5/main.java
class ProblemFive {
    public static void main(String[] args) {
        int i = 1;
        while (true) {
            if (isEvenlyDivisible(i)){
                break;
            }
            i++;
        }
        System.out.println(i);
    }
    static boolean isEvenlyDivisible(int n){
        for (int i = 1; i < 21; i++) {
            if (n % i != 0){
                return false;
            }   
        }
        return true;
    }
}